src/main/java/duke/data/Storage.java

```diff
@@ -0,0 +1,77 @@
+package duke.data;
+
+import duke.task.*;
```

Comment:
Imported classes should always be listed explicitly even though in this case, your Storage class utilizes all the classes in your task package as it gives an excellent documentation value for the class at hand and makes the class easier to comprehend and maintain.